recipes/nansat/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nansat" %}
+{% set version = "0.6.17" %}
+{% set sha256 = "ecaa305250d3a34ad7d4922034bf33731ec0da127afa533cbb74728a69bab2dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+ fn: v{{ version }}.tar.gz
+ url: https://github.com/nansencenter/nansat/archive/v{{ version }}.tar.gz
+ sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [osx or not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - pythesint
+    - numpy 1.11.3
```

Comment:
This package only works with `numpy 1.11.3`?

Reply:
No, but this is the combination of versions that is tested to work properly on all platforms. Might be other combination that are good as well, but there are limitations as scipy, gdal and matplotlib all require numpy. Matplotlib being the most difficult of these. It is possible that we could build it with these, but run it with a newer version, but that has not been tested properly.

Reply:
You can build with numpy 1.11.3 and let the run requirements be `numpy >=1.11.3`. Also you can relax other versions like scipy, matplotlib, etc. 

>  this is the combination of versions that is tested to work properly on all platforms

This recipe is tested with `nosetests nansat`. Is that enough to be able to say that it is tested to work or do you run more tests to make sure it works on a platforms

Reply:
We run other integration tests as well. But I am testing to run them now with the latest version of these libraries, while still building with the strict requirements.

The problem I foresee is if there is something in a new release that breaks backward compatibility. I assume the feedstock for Nansat will not rebuild each time a dependency has updated, hence nosetests will not be tested for each new dependency update.

Do you not think this can be a possible issue?

Reply:
I've now eased up on most run requirements, except from scipy, which has to be 0.18.1.

Reply:
If it really doesn't work with versions of scipy other than `0.18.1`, fine, but it's going to be a hassle for people trying to use your package. (You're correct that new versions won't rerun the tests, but most packages don't break backwards compatibility very often.)

Reply:
I'm aware of the hassle, I'm having enough of it myself :)

It is on the list to update this strict requirement. I'm not sure exactly what the issue it, but I believe they changed some round-off stuff behaviour that broke backwards compatibility.